Frontend/auction-app/src/index.css

```diff
@@ -1,13 +1,65 @@
-body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-}
-
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
-    monospace;
+.form-box {
+  display: flex;
+  justify-content: center;
+}
+
+.form {
+  width: 702px;
+  height: 830px;
```

Comment:
We should aim for responsiveness when writing css classes.
Having a concrete width and height is not the way to go, as the sizes will always be constant.
Of course, we should set limits to width and height properties.
This is why we use min/max height/width just to keep the object from over-stretching or becoming too small.

Reply:
If we need to use width, it should be used in percentages.